### PR TITLE
The ES pod went to Error state and after few hours the elasticsearch containers not running in the ES pods

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1752815#c15

https://coreos.slack.com/archives/CB3HXM2QK/p1575473699343800

### CODE REVIEW

**Comment by mburke5678** [comment]
@jcantrill PTAL

**Review by jcantrill — APPROVED** [review]

**Comment by mburke5678** [comment]
@Shrivaibavi Let me know if these changes address your concerns.

**Comment by mburke5678** [comment]
@anpingli PTAL

**Review by anpingli — COMMENTED** [review]
LGTM

File: modules/cluster-logging-elasticsearch-limits.adoc [hunk 1]
```diff
@@ -34,11 +37,10 @@ spec:
       elasticsearch:
         resources: <1>
           limits:
-            cpu: "4000m"
-            memory: "4Gi"
+            memory: "16Gi"
           requests:
-            cpu: "4000m"
-            memory: "1Gi"
+            cpu: "1"
```
Comment (portante) [inline modules/cluster-logging-elasticsearch-limits.adoc]: Why are we reducing the number of available CPUs for Elasticsearch to `1` by default?

Comment (jcantrill) [reply modules/cluster-logging-elasticsearch-limits.adoc]: The default provided by the operator is 1 and has been since 3.9

Comment (portante) [reply modules/cluster-logging-elasticsearch-limits.adoc]: That seems pretty bad.  How is the default changed?

**Pull request merged**